src/registry.ts

```diff
@@ -97,5 +97,5 @@ export async function setEntry(
   if (response.status === 204) {
     return true;
   }
-  throw new Error(`unexpected response status code ${response.status}`);
+  return false;
```

Comment:
`setEntry` should probably be just. In other case it fails, let's say because revision was wrong, user would not get any error message and would not know how to act - either retry or change the payload, returning `false` doesn't tell user anything useful.

```javascript
return this.executeRequest({
           ...opts,
      	      method: "post",
      	      data: {
      	        publickey: {
                          algorithm: "ed25519",
          	          key: Array.from(publickey),
       	        },
        	        datakey,
        	        revision: entry.revision,
        	        data: Array.from(Buffer.from(entry.data)),
        	        signature: Array.from(signature),
	      },
});
```